### PR TITLE
Add Amazon EC2 c7g instance configuration

### PR DESCRIPTION
**Pull request by mifisignal**
Signed-off-by: Michael Fischer <fiscmi@amazon.com>

Add support for Amazon EC2 c7g instances.

```release-note
Add support for Amazon EC2 c7g instances
```

### CODE REVIEW

**Review by christarazi — APPROVED** [review]

**Review by borkmann — APPROVED** [review]

File: pkg/aws/eni/limits/limits.go [hunk 1]
```diff
@@ -121,6 +121,15 @@ func populateStaticENILimits() {
 		"c6gn.large":    {Adapters: 3, IPv4: 10, IPv6: 10},
 		"c6gn.medium":   {Adapters: 2, IPv4: 4, IPv6: 4},
 		"c6gn.xlarge":   {Adapters: 4, IPv4: 15, IPv6: 15},
+		"c7g.12xlarge":  {Adapters: 8, IPv4: 30, IPv6: 30},
```
Comment (sayboras) [inline pkg/aws/eni/limits/limits.go:124]: out of curiousity, how do get details for this new instance types ? I am checking lastest CLI command, but c7g is not available in output. Thanks.

https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-instance-types.html

Comment (sayboras) [inline pkg/aws/eni/limits/limits.go:124]: out of curiousity, how do get details for this new instance types ? I am checking lastest CLI command, but c7g is not available in output. Thanks.

https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-instance-types.html

Comment (sayboras) [inline pkg/aws/eni/limits/limits.go:124]: out of curiousity, how do get details for this new instance types ? I am checking lastest CLI command, but c7g is not available in output. Thanks.

https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-instance-types.html

Comment (sayboras) [reply pkg/aws/eni/limits/limits.go:124]: (sorry about multiple comments with same content, seems like github was having a short issue)

Comment (sayboras) [reply pkg/aws/eni/limits/limits.go:124]: I didn't notice your email, so all good :100:

**Comment by borkmann** [comment]
@christarazi / @sayboras Do we need to run ci-eks or not affected and we can mark this one as ready-to-merge?

**Comment by pchaigno** [comment]
ConformanceEKS is passing and reviews are in. Marking ready to merge.

**Pull request merged**